corehq/apps/users/migrations/0024_add_app_access_permissions.py

```diff
@@ -0,0 +1,38 @@
+from django.db import migrations
+
+from dimagi.utils.couch.database import iter_docs
+
+from corehq.apps.users.models import UserRole
+from corehq.util.django_migrations import skip_on_fresh_install
+
+
+@skip_on_fresh_install
+def _migrate_web_apps_permissions_to_app_permissions(apps, schema_editor):
```

Comment:
do you have a sense of how long this will take on any envs? I'm just wondering if it's ok to run as part of deploy if it needs to follow a different process: https://commcare-hq.readthedocs.io/migration_command_pattern.html#auto-managed-migration-pattern

Reply:
I am not particularly sure about that but @orangejenny mentioned that we have run this sort of migrations in past.

Do you think that keeping them separate from deploy would be a good thing to do @snopoke?

Reply:
That's a good point, this should ideally be run separately from deploy. Recent permissions migrations have taken around 15 minutes and have been run during deploy, but we got complaints from customers, because they noticed uploads weren't running during that time.

Reply:
Cool. Will run migrations separately then.
Thanks for your inputs @orangejenny and @snopoke